### ISSUE TITLE
LTN 2.0 

### ISSUE THREAD

**Issue by cablegunmaster** [Yousei9/Logistic-Train-Network#349]
**TL;DR**
Re-vitalize LTN, there is a working port on PSR-J0740 Branch.
After this what's next? 

**What ?**
A new version on 2.0 on the mod page of factorio; first its just being a ported version and later on a new version to remodel functionality;

Suggestions;
- Trains can always go somewhere else, unless its low on fuel ( like less as 10 in a fuel slot of a train) .
- Train can wait at unloader unless another train is required to be at that stop. ( no idea how to figure this one out yet) but seems like a fun puzzle.
- Train only goes to depot if its pushed out of its slot from an unloader and no other orders are required from this train.
- Train does not only need to be at a depot to get new orders, but needs to be at an unloader station.
- Train refuels if low on fuel.

**Why ?**
WHY do you think it increases the value of LTN?

Well we can go 2 routes, the mod can be marked as doomed or it can be marked as a spot of collaboration between different developers, who are willing to change the mod and alter it to something new.

**Communication**
I talked so far with PSR_J0740 and Kryojenik (owner of Modernized combinator)  

- PSR_j0740 is wondering what to do now he has it ported to a new version. 
- Kryojenik is awaiting a new version before something is ported for a modernized combinator.

**Conclusion**
We can mark LTN as deprecated or we can build it out with more functionality , rebuild it make a few ideas on how it should work make some schematics on how train should behave and implement it.

**Comment by PSR-J0740** [comment]
Optera,

I’ve ported LTN to Factorio 2.0, as it’s clear the community wanted a port for this mod and would greatly benefit from its continued functionality. I wanted to reach out to request your permission to redistribute the ported version.

If you have any input or specific conditions regarding this, I’d be happy to accommodate them. Your work has been invaluable to the Factorio modding scene, and I’d like to ensure the mod remains accessible.

Thank you again for all the effort you put into creating LTN.

**Comment by 0ptera** [comment]
Please read the summary in #350 .

I don't want to drag outdated concepts to 2.0 with a rudimentary port.
